spec/features/inactive_families/admin_views_all_inactive_families_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "Admin can see all the inactive families when in families#index" do
```

Comment:
Again, suggesting to rename to something more accurate: "Admin sees inactive families only"

Reply:
Done in sha: `f92ad52`